cmake/External_SDL.cmake

```diff
@@ -0,0 +1,20 @@
+message(STATUS "FetchContent: SDL")
```

Comment:
I think this folder should be named vendor or thirdparty to indicate that these dependencies are provided by other people/organizations. In this case we can safely drop the "External" prefix from the names of macros as the folder name becomes a kind of namespace.

Reply:
Also, we may include additional dependencies (e.g. libfmt as a header) so the folder will be used anyway.